src/components/booksList/booksList.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable arrow-body-style, max-len, react/forbid-prop-types */
```

Comment:
## Move the bookList component
- [ ] The `bookList.js` component needs to be moved to the `container` directory. You only need to have the `App.js` and the `book.js` components in this directory.